lesson-4/script.js

```diff
@@ -0,0 +1,74 @@
+const correctResult = (0.1 + 0.2).toFixed(1);
```

Comment:
`.toFixed` will return result in a number
```suggestion
const correctResult =+=(0.1 + 0.2).toFixed(1);
```

Reply:
тут я не зрозумів рішення. можна детальніше або посиланням на інфо?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed